package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "uptime-kuma",
-    "version": "1.11.3",
+    "version": "1.11.4",
```

Comment:
Why?

Reply:
I thought it is necessary that the database patch is working, but tested it now without version change and it works also.

I reverted the version change.

I also found some other issues in my code and corrected them.